ITI/TF/Volume2/ITI-18.html

```diff
@@ -2033,26 +2088,56 @@ <h6 id="3.18.4.1.2.4">3.18.4.1.2.4 Stored Query IDs</h6>
-                    on the Document Registry. If this parameter is specified, and the Document Registry does not support
-                    it, the Document Registry shall ignore. If it is specified, and the Document Registry does support
-                    it, the proper information is returned.</p>
-                <ul>
-                    <li class="bullet-list1">If the Document Consumer does not support the On-Demand Documents Option,
-                        it will send a Registry Stored Query request which does not contain the $XDSDocumentEntryType
-                        parameter. The Document Registry will therefore not supply any On-Demand Document Entries in the
-                        query response.
-                    </li>
-                    <li class="bullet-list1">If the Document Consume does support the On-Demand Documents Option<, then
-                        it will be able to specify the $XDSDocumentEntryType parameter containing the uuid for On-Demand
-                        Document Entries in a Registry Stored Query. A Document Registry with the On-Demand Documents
-                        Option will recognize the $XDSDocumentEntryType parameter and process accordingly. A Document
-                        Registry which does not support the On-Demand Documents Option will ignore the
-                        $XDSDocumentEntryType parameter. Since there cannot be any On-Demand Document Entries held by a
-                        Document Registry which does not support On-Demand Documents, this is a consistent response to
-                        the request.
-                    </li>
-                </ul>
+                <p>Compatibility of Options supported by Actors are defined as follows:</p>
+                <ol>
+                    <li class="numbered-list1">
+                        <p>The presence or absence of the optional $XDSDocumentEntryType parameter triggers different behaviors
+                        on the Document Registry. If this parameter is specified, and the Document Registry does not support
+                        it, the Document Registry shall ignore. If it is specified, and the Document Registry does support
+                        it, the proper information is returned.</p>
+                        <ul>
+                            <li class="bullet-list1">If the Document Consumer does not support the On-Demand Documents Option,
+                                it will send a Registry Stored Query request which does not contain the $XDSDocumentEntryType
+                                parameter. The Document Registry will therefore not supply any On-Demand Document Entries in the
+                                query response.
+                            </li>
+                            <li class="bullet-list1">If the Document Consume does support the On-Demand Documents Option<, then
```

Comment:
as written in the cp change this line in:
If the Document Consumer does support the On-Demand Documents Option,